README.md

```diff
@@ -158,7 +158,7 @@ NOTE: The steps need to be done in order to create a new Hyrax based app.
 Generate a new Rails application using the template.
 
 ```
-rails _5.2.4.4_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v2.9.3/template.rb
+rails _5.2.4.4_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v2.9.4/template.rb
```

Comment:
Is it worth making this rails 5.2.4.5?
```suggestion
rails _5.2.4.5_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v2.9.4/template.rb
```

Reply:
i think that's a different PR?

Reply:
Good point.  It should probably go in step with the CircleCI config.